app/helpers/application_helper.rb

```diff
@@ -195,4 +195,29 @@ def logo_svg
   def community_qualified_name
     "The #{ApplicationConfig['COMMUNITY_NAME']} Community"
   end
+
+  def truncate_html(html, length)
+    html_without_tags = strip_tags(html)
+    tags_length = html.length - html_without_tags.length
+    return html if html_without_tags.length <= length
+
+    Nokogiri::HTML::DocumentFragement.parse(html.truncate(length + tags_length)).to_xml
+  end
+
+  def sponsorship_credits_price(level)
```

Comment:
Mmm I think this reappeared... because of an error with rebasing?

Reply:
I think happened after I resolved a conflict on the file.

Reply:
Got it, it can be safely removed :)

Reply:
But I also just made a change to the function to use case instead of if..elsif because of the codeclimate errors

Reply:
@bolariin that's nice but the function is not used anywhere, it doesn't exist anymore on master, that's why I said it came back :D

Reply:
(if you check your PR's diff you'll notice this is marked as an entirely new function)

Reply:
so I should just remove it then. I understand now. Thanks